### PR TITLE
Add validation between unit type and address format when adding property

### LINKED ISSUE #207: Mismatch between address format and unit type
Examples of mismatch:
Unit Type: `HDB 3-Room`; Address: `25 Kent Ridge Road, S123456` (HDB should have unit level and number: `#XX-XX`)
Unit Type: `Landed Property`; Address: `25 Kent Ridge Road #01-23, S123456` (Landed Property should not have unit level/number)

### PR DESCRIPTION
**Pull request by OVReader**
Fix #207

### CODE REVIEW

File: src/main/java/seedu/duke/Messages.java [hunk 1]
```diff
@@ -122,6 +122,18 @@ public class Messages {
...
+            + "Certain unit types will require specific address format. Please refer to the description below:\n"
+            + "  1. Unit type with <LP> must not have #[unit level]-[unit number] in address.\n"
+            + "     Format:\n"
+            + "       [BLOCK NUMBER] [STREET NAME], S[POSTAL CODE]\n"
+            + "  2. Unit type without <LP> must have #[unit level]-[unit number] in address.\n"
+            + "     Format:\n"
+            + "       [BLOCK NUMBER] [STREET NAME] #[unit level]-[unit number], S[POSTAL CODE]\n"
+            + "       [BLOCK NUMBER] [STREET NAME] #[unit level]-[unit number] [building name], S[POSTAL CODE]\n"
+            + "Note: HDB Terrace House (special case) is not restricted by any format.";
+
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Messages.java:146]: Good elaboration on the error message.

File: src/main/java/seedu/duke/parsermanager/ParseAddProperty.java [hunk 2]
```diff
@@ -106,7 +107,11 @@ public class ParseAddProperty extends ParseAdd {
 
     private static HashMap<String, String> UNIT_TYPE_HASHMAP;
 
-
+    private static final String HASH_SYMBOL = "#";
+    private static final String LANDED_PROPERTY_TAG = "LP";
+    public static final String HDB_TAG = "HDB";
+    public static final String CONDOMINIUM = "Condominium";
+    public static final String PENTHOUSE = "Penthouse";
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/parsermanager/ParseAddProperty.java]: Would be good if u can partition these constants and insert meaningful comments :D

**Review by wilsonngja — COMMENTED** [review]
Overall LGTM. Good job :D

File: src/main/java/seedu/duke/Messages.java (hunk 1 above)
Comment (ngdeqi) [reply src/main/java/seedu/duke/Messages.java:146]: Agree, clear message with good formatting

**Review by ngdeqi — COMMENTED** [review]
Variable names are good, can't find any violations of code quality. LGTM!

**Review by FeliciaBeatrice — APPROVED** [review]
LGTM

**Pull request merged**